packages/okta-react-native/setupJest.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+global.XMLHttpRequest = global.XMLHttpRequest || function() {};
+global.fetch = jest.fn();
```

Comment:
Why are XHR and fetch treated differently?  You want XHR requests to actually happen, but not fetch ones?

Reply:
I just copied the setupJest file LOL, I don't even think this is needed since there weren't any network calls within the module.

Reply:
I think this is actually needed for jest to setup. It is required to correctly polyfill react-native

Reply:
but...why?  fetch() is mocked, but XHR is treated differently?

Reply:
I just changed XMLHttpRequest to a jest mocked function to keep it consistent. Not sure why it was like that lol.